content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -506,18 +534,24 @@ If W&B encounters errors accessing the bucket or detects invalid settings, an er
 1. Provide a name for the team.
 1. Set **Storage Type** to **External storage**.
 1. Click **Bucket location**.
-1. To use an existing bucket, select it from the list. To add a new bucket, click **Add bucket** at the bottom, then provide the bucket's details.
+1. To use an existing bucket, select it from the list.
+1. To create a new bucket, click **Add bucket** at the bottom, then:
+    1. Click **Cloud provider** and select **CoreWeave**, **AWS**, **GCP**, or **Azure**.
+    1. Enter the bucket details:
+        - **Name**: For all providers except Azure, enter the bucket name. For Azure, this is the container name.
+        - **Path** (optional): Enter a sub-path to use within the bucket.
+    1. Provide additional connection settings for the chosen cloud provider:
+        - **CoreWeave**: No additional settings required.
+        - **AWS**: Optionally provide a **KMS key ARN** for encryption.
+        - **GCP**: No additional settings required.
+        - **Azure**: 
+            - **Account name** (required): The Azure storage account name.
+            - **Tenant ID** (optional): The Azure Active Directory tenant ID.
+            - **Managed Identity Client ID** (optional): The client ID for managed identity authentication.
+    
+    {{% alert %}}When you click **Create team**, W&B will automatically create the bucket in your cloud provider with the specified configuration.{{% /alert %}}
 
-    Click **Cloud provider** and select **CoreWeave**, **AWS**, **GCP**, or **Azure**.
-1. Specify the bucket details.
-    - For **CoreWeave**, provide only the bucket name.
-    - For Amazon S3, GCP, or S3-compatible storage, provide the full bucket path you [determined earlier](#determine-the-storage-address).
-    - For Azure on W&B Dedicated or Self-Managed, set **Account name** to the Azure account and **Container name** to the Azure blob storage container.
-    - Optionally:
-      - If applicable, set **Path** to the bucket sub-path.
-      - **AWS**: Set **KMS key ARN** to the ARN of your KMS encryption key.
-      - **Azure**: If applicable, specify values for **Tenant ID** and **Managed Identity Client ID**.
-     - Invite members to the team. In **Invite team members**, specify a comma-separated list of email addresses. Otherwise, you can invite members to the team after it is created.
+1. Invite members to the team. In **Invite team members**, specify a comma-separated list of email addresses. Otherwise, you can invite members to the team after it is created.
 1. Click **Create team**.
 
 If W&B encounters errors accessing the bucket or detects invalid settings, an error or warning displays at the bottom of the page. Otherwise, the team is created.
```

Comment:
Line 568 "For example: `cw://bucket-name.cwobject.com` is correct, while ~`cw://cwobject.com/bucket-name/`~ is not." needs an edit. It shows up as a strikethrough which is not legible on the docs:

<img width="992" height="62" alt="Screenshot 2025-09-04 at 8 22 56 AM" src="https://github.com/user-attachments/assets/affe912d-70a7-4aa5-a442-4140d40f38eb" />

Reply:
Done. Good catch. This must have been introduced in v0.70.